example/server.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path')
```

Comment:
what's the need of creating a separate `node server` and `webpack`, when we are already making use of `rollup`? Read about [Rollup watch](https://rollupjs.org/#rollup-watch)

Reply:
I think our demo's javascript script depended online version. And I think this dev server is depend our code in src. It is convenient to debug and develop. What do you think? I think demo should update so I created example.

Reply:
we can simply reuse demo for testing purposes as well

Reply:
you have to change the js cdn. In my opinions, it is not necessary. Just run `npm run dev` and then see the demo and testing is very cool!

Reply:
@stephenkingsley will be happy to merge if you reuse existing demo code and integrate in `rollup` as advised above

Reply:
Ok, I think @sangle7 can do this! Reuse existing demo and delete example folder.

Reply:
@stephenkingsley closing this pull request for now

Reply:
@ApoorvSaxena Ok, I will open another pr